docs/registry/_index.md

```diff
@@ -40,6 +41,21 @@ See [Machine Learning](/ml/) for more information.
 The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
 You can also [upload your own model to the registry](/ml/upload-model/).
 
+You can search the available ML models from the Viam registry here:
+
```

Comment:
@npentrel I applied all your suggestions and it is almost ready, but the styling on this table is looking wild and I haven't been able to troubleshoot. Any ideas? I tried looking in "modellist" css rules and I can't tell if anything is acting on the table columns